multicluster/Makefile

```diff
@@ -50,6 +50,12 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: test
+test:  
```

Comment:
you should name it as test-integration, please refer to Makefile in antrea root directory, should be similar like this part https://github.com/antrea-io/antrea/blob/main/Makefile#L216-L221